INSTALL.md

```diff
@@ -1,30 +1,58 @@
 ## A- Local Installation using pypi
 
-Using pypi repository, you can install mantisshrimp and its
-dependencies:
+There are 3 ways to install mantisshrimp and its dependencies using `pip install`. 
 
-Install PyTorch as per your preference from
-[here](https://pytorch.org/get-started/locally/#start-locally).
+> **Note**: You can check out the following blog post: [3 ways to pip install a package ](https://ai-fast-track.github.io/blog/python/2020/03/17/how-to-pip-install-package.html) for more a detailed explantion on how to choose the most convenient option 
+for you. 
 
-Installing fastai and/or Pytorch-Lightning packages
+
+### Option 1: Installing from pypi repository
+ 
+#### All Packages
+To install mantisshrimp package and both Fastai and Pytorch-Lightning libraries, run the following command:
 
 ```bash
-pip install fastai2
-pip install pytorch-lightning
+pip install mantisshrimp[all]
 ```
 
-Installing albumentations package
+#### Mantisshrimp + Fastai
+To install mantisshrimp package and only the Fastai library, run the following command:
 
 ```bash
-pip install albumentations
+pip install mantisshrimp[fastai]
```

Comment:
Should we add a comment that this is only going to be available in a few days?

Maybe point to git install for now

Reply:
Or to local install, since I think most users right now are also contributors

Reply:
Change it to
git clone  --depth=1 mantisshrimp
the repo. pip install .[all] for now.

Depth 1 will limit the .git thingie for end user.

Reply:
Thank you @oke-aditya for reminding me. Yesterday, I read tweets from Jeremy and Hamel (Github) suggesting that, and then I forgot to added it.

Reply:
Ow, the git thingy is 200mb, I don't think it's a problem for development.